CHANGELOG.rst

```diff
@@ -4,10 +4,12 @@ Changelog
 This document describes changes between each past release.
 
 
-2.11.0 (unreleased)
+2.10.1 (2015-11-03)
 -------------------
 
-- Nothing changed yet.
+**Bug fixes**
+
+- Make sure read enpoint are activated in readonly mode. (#539)
```

Comment:
nit: endpoints

nit: Give detail ? Fix comparison of enpoints methods to make sure [...]